### PR TITLE
Sets the wizard shipping address value to 'always'

### PR DESCRIPTION
**Pull request by alexmigf**

### CODE REVIEW

**Comment by alexmigf** [comment]
Now mimics the document display settings:

Captura de ecrã de 2022-09-23 15-16-51

File: includes/views/setup-wizard/display-options.php [hunk 1]
```diff
@@ -10,47 +10,83 @@
...
+				<label for="display-number" class="checkbox"><?php esc_html_e( 'Invoice number', 'woocommerce-pdf-invoices-packing-slips' ); ?></label>
 			</th>
 			<td>
-				<label for="display-number" class="checkbox"><?php esc_html_e( 'Display invoice number', 'woocommerce-pdf-invoices-packing-slips' ); ?></label>
+				<input type="hidden" name="wcpdf_settings[wpo_wcpdf_documents_settings_invoice][display_number]" value="">
+				<select id="display-number" name="wcpdf_settings[wpo_wcpdf_documents_settings_invoice][display_number]">
+					<?php
+						$options = array(
+							''				=> __( 'No' , 'woocommerce-pdf-invoices-packing-slips' ),
+							'invoice_date'	=> __( 'Invoice Date' , 'woocommerce-pdf-invoices-packing-slips' ),
```
Comment (YordanSoares) [inline includes/views/setup-wizard/display-options.php:81]: @alexmigf I think here should be, right?:
`'invoice_number'	=> __( 'Invoice Number' , 'woocommerce-pdf-invoices-packing-slips' ),`

File: includes/views/setup-wizard/display-options.php [hunk 2]
```diff
@@ -10,47 +10,83 @@
...
+							'order_date'	=> __( 'Order Date' , 'woocommerce-pdf-invoices-packing-slips' ),
```
Comment (YordanSoares) [inline includes/views/setup-wizard/display-options.php:82]: In the same way here:
`'order_number'	=> __( 'Order Number' , 'woocommerce-pdf-invoices-packing-slips' ),`

**Review by YordanSoares — CHANGES_REQUESTED** [review]

**Review by YordanSoares — APPROVED** [review]
## Settings in the `main` branch:
image

## Settings improved with this PR:
image

## Test performed:
| Setting | Result description | Is working?
| --- | --- | --- | 
| Shipping address | Select field: All the options are applied correctly | ✔️ |
| Email address | Checkbox: If checked, it is applied | ✔️ |
| Phone number | Checkbox: If checked, it is applied | ✔️ |
| Invoice date| Select field: All the options are applied correctly | ✔️ |
| Invoice Number| Select field: All the options are applied correctly | ✔️ |

**Pull request merged**